synapse/models/telco.py

```diff
@@ -193,6 +193,11 @@ def getModelDefs(self):
                     'doc': 'A mobile cell site which a phone may connect to.'
                 }),
 
+                # TODO - eventually break out ISO-3 country code into a sub
+                # https://en.wikipedia.org/wiki/TADIG_code
+                ('tel:mob:tadig', ('str', {'regex': '^[A-Z0-9]{5}$', 'strip': 1}), {
```

Comment:
I do see we have a few other places we use `1` instead of `True` but I'm going to clean those up in the 3x branch right now for consistency.
```suggestion
                ('tel:mob:tadig', ('str', {'regex': '^[A-Z0-9]{5}$', 'strip': True}), {
```

Reply:
Thanks. Copy/pasta error ;)